doc/digraph.xml

```diff
@@ -252,6 +252,89 @@ gap> DigraphVertexLabel(gr, 2);
 </ManSection>
 <#/GAPDoc>
 
+<#GAPDoc Label="DigraphEdgeLabels">
+<ManSection>
+  <Oper Name="DigraphEdgeLabels" Arg="digraph"/>
+  <Oper Name="SetDigraphEdgeLabels" Arg="digraph, labels"/>
+  <Oper Name="SetDigraphEdgeLabels" Arg="digraph, func"/>
+  <Description>
+    If <A>digraph</A> is a digraph, then <C>DigraphEdgeLabels</C> returns a
+    copy of the labels of the edges in <A>digraph</A> as a list of lists
+    <C>labels</C> such that <C>labels[i][j]</C> is the label on the edge
+    from vertex <A>i</A> to vertex <A>j</A>.
```

Comment:
I'm not sure what change you're proposing.

Which problems will come up, and what do you mean by "using an edge"?

I realise that, with your PR, one is able to set the edge label of `[i,j]` using `SetDigraphEdgeLabel(gr, i, j, value);`, and one can read it by using `DigraphEdgeLabel(gr, i, j);`. I think that's a good idea. But currently, to read/write *all* edge labels at once, you don't do it by explicitly referring to both ends of each edge.

Are you suggesting that `DigraphEdgeLabels` should return a square matrix, so that `labels[i][j]` has the  behaviour you've documentation above? I can't think what else you might mean. One question then is, what value should the matrix have when there is no edge?